internal/exec/stages/files/files.go

```diff
@@ -30,6 +32,9 @@ import (
 
 const (
 	name = "files"
+
+	// see https://github.com/systemd/systemd/pull/11131/files#diff-9e1684efe63d10844b8dd6708fc60e8dR414
+	relabelFilePath = "/run/systemd/relabel-extra.d/ignition.relabel"
```

Comment:
Perhaps it would make sense to keep the systemd path as const and add `ignition.relabel` just where we use it?